### ISSUE TITLE
Add coverage for cli contract bump and restore

### ISSUE THREAD

**Issue by sreuland** [stellar/system-test#70]
### What problem does your feature solve?
no coverage for soroban cli `contract bump` or `contract restore`

### What would you like to see?
new test coverage with steps/scenarios that execute cli `contract bump` and `contract restore` and verify the expected cli output.

**Comment by sreuland** [comment]
note, this only establishes coverage for js, cli coverage of the same functionality will be done as integration tests between cli and rpc mainteained in soroban-tools repo, should link here for reference.

**Comment by Shaptic** [comment]
Footprint bumps are separate from autorestore; they're just the explicit way to bump ledger keys. `RestoreFootprintOp` is still needed in the edge case where autorestore exceeds network limits. I'm writing integration tests in RPC to test this case https://github.com/stellar/stellar-rpc/pull/481/, but we could also do that from the JS side here the same way.

**Comment by sreuland** [comment]
> Footprint bumps are separate from autorestore; they're just the explicit way to bump ledger keys. RestoreFootprintOp is still needed in the edge case where autorestore exceeds network limits. 

Yes, this was targeting at a level that is too low. Have rewritten criteria to be at a higher level of user interaction with state expiration touch points. Thanks!
